### DISCUSSION TITLE
Install fails

### DISCUSSION THREAD

**Discussion by 1noob4ever** [Security-Onion-Solutions/securityonion#3125] [Unsupported Versions]
Not sure why this is happening. When I check eth0 it is set to the ip I'm using in the setup questions. ???

![image](https://user-images.githubusercontent.com/43621515/109080612-86141f80-76c6-11eb-815c-d3a9b405010a.png)

**Comment by weslambert** [comment]
Are you using DHCP?  There could be something else cached, so I'd try double-checking via nmcli and ifcfg files to ensure you don't have anything weird going on.

**Comment by TOoSmOotH** [comment]
2.3.30 will allow you to continue on from this screen and try and install. Like Wes said this usually happens when you have multiple dhcp interfaces connected or you had a dhcp address and switch to a static IP.

**Reply by 1noob4ever** [reply]
I have 4 monitoring nics and 2 of them were given ip's by dhcp. The management nic was set with a static. Is 2.3.30 available?
I had considered installing with only 2 nics active and installing additional nics later but is there away around this issue?
